organization/roadmap.md

```diff
@@ -0,0 +1,66 @@
+---
+type: Resource
+---
+
+# Roadmap
+
+This roadmap serves as a model for how we're developing, growing and scaling our organization. It sets out the roles for both the members and the central organization they control.
+
+## Phase 0: get the organization founded
+
+Foundation For Public Code:
```

Comment:
```suggestion
Our organization:
```

Reply:
I would do this when we change the name everywhere, this will make it easier to do and clearer for now.

Reply:
@MirjamvT found this to be too confusing since who is 'us' vs 'them', it is worth giving  this some more thought.

Reply:
Maybe use of the word 'team' might help too? so there is the team that needs to do things and the members/the organization that needs to do things.
(I am trying to see the members as the organization)

headers might be:
Team of the foundation 
or
Executive team of the Foundation
or, without 'team'
Executive Organization

and the members would be onder the header:

Founding members
and later (phase 3)
Members
or
The organization's members

This needs more thought, still.

Reply:
resolved by 'staff of' and 'members of' the foundation